src/apis/reservation/queries.ts

```
@@ -1,14 +1,39 @@
-      default:
-        throw new Error(response.message);
+  try {
+    const response = await httpClient.get<Reservation>(`/reservation/${orderNumber}`);
+
+    if (!response.success) {
+      switch (response.status) {
+        case 401:
+          redirect(ROUTES.LOGIN);
+        case 404:
+          notFound();
+        default:
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)